### PR TITLE
Performance audit

### PR DESCRIPTION
**Pull request by bilalrajput09**
Enhanced database performance.
Used EXPLAIN ANALYZE to check the performance.

### QUERIES:

EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animal_id = 4;
EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
EXPLAIN ANALYZE SELECT * FROM owners where email = 'owner_18327@mail.com';

 
 BEFORE
Screenshot 2023-06-19 133452
AFTER
Screenshot 2023-06-19 133925
BEFORE
Screenshot 2023-06-19 134807
AFTER
Screenshot 2023-06-19 134918
BEFORE
Screenshot 2023-06-19 135048
AFTER
Screenshot 2023-06-19 135207

### CODE REVIEW

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @bilalrajput09 , @Ofarouq310  and @tomdan-ai , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that uses the wrong project URL instead of the `Create your own Enumerable` project you have submitted `performance audit`, kindly ensure you submit the correct project

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: queries.sql [hunk 1]
```diff
@@ -203,6 +203,10 @@ GROUP BY s.name
 ORDER BY num_visits DESC
 LIMIT 1;
 
+EXPLAIN ANALYZE SELECT COUNT(*) FROM visits where animals_id = 4;
+EXPLAIN ANALYZE SELECT * FROM visits where vet_id = 2;
```
Comment (mukhammadrizooff) [inline queries.sql:207]: - [ ] [REQUIRED] Good job so far! The second `screenshot` should show how much time the query takes `AFTER` the improvements introduced by you - The `execution` time should be significantly `lower` than 1 sec (`1000 ms`). After the implementation, please `update` the screenshot with the `AFTER` result on your `Pull Request` summary 🙏🏼 

image

**Review by mukhammadrizooff — CHANGES_REQUESTED** [review]
Hi @TEAM, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (bilalrajput09) [reply queries.sql:207]: FIXED 😊.

## LATEST SNAPSHOTS: 

**BEFORE:**
Screenshot 2023-06-19 134918
**AFTER:**
latest_execution_time

**Review by mukhammadrizooff — APPROVED** [review]
Hi @TEAM, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 

## Highlights
- Correct GitFlow ✔️ 
- Good implementations ✔️
- Descriptive `README` file ✔️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**